app/views/prison/visits/_visitor_details.html.erb

```diff
@@ -24,7 +24,7 @@
     <% end %>
   </ul>
   <div class="form-group">
-    <% if !@visit.contact_list_unknown? %>
+    <% if Nomis::Feature.contact_list_enabled?(@visit.prison_name) && !@visit.contact_list_unknown? %>
```

Comment:
There is an argument that `Nomis::Feature.contact_list_enabled?(@visit.prison_name) && !@visit.contact_list_unknown?` could be encapsulated into a helper as it's used at least 3 times in this view.